pages/linux/nmcli-monitor.md

```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor NetworkManager showing changes with connection status and similar.
+> More information: <https://developer.gnome.org/NetworkManager/stable/nmcli.html>.
+
+- Monitor NetworkManager changes:
+
+`nmcli monitor`
```

Comment:
```suggestion
`nmcli monitor`

- List connection profiles:

`nmcli connection show`
```

Reply:
This page is about `nmcli monitor` though, and the new command is `nmcli connection` 🤔

Reply:
I became too enthusiatic when i saw `nmcli` .. i use this `connection` subcommand often... :( my bad

Reply:
We'll get to it, just remember to bring it up if I forget when revamping the nm-connect page